### ISSUE TITLE
Missing about section on Banner pattern page

### ISSUE THREAD

**Issue by gldgrnt** [wmcadigital/wmn-design-system#277]
**Describe the bug**
AXE is showing 'Heading levels should increase by one'. [https://designsystem.wmnetwork.co.uk/patterns/banner/](https://designsystem.wmnetwork.co.uk/patterns/banner/)

**Fix**
Add an about section, like the rest of the subpages have, with an `<h2>`

**Comment by daylesalmon** [comment]
Relates to #287
